doc/CHANGES.md

```diff
@@ -9,6 +9,20 @@
 * Drop support for Debian 9 Stretch (Graeme Porteous)
 * Drop support for Debian 10 Buster (Graeme Porteous)
 * Drop support for Ubuntu 19.04 Bionic (Graeme Porteous)
+* Add support `ActiveStorage` for raw emails (Graeme Porteous)
+
+## Update Notes
+
+* There are some database structure updates so remember to run
+  `bin/rails db:migrate`
+* This release is the last to support the old raw emails file store. Please now
+  configure `ActiveStorage`. This will allow you to use an on disk store as
+  previously, but also opens the options for using cloud backed storage
+  providers. Copy `config/storage.yml-example` to `config/storage.yml` and edit
+  to configure your storage.
```

Comment:
The defaults should Just Work as before, so probably want to make that explicit here – you only need to edit if you're doing something custom?

Reply:
Improved in 8153c0338